src_blog/_posts/New-Release-Grain-v0.4.0-Cereal.md

```diff
@@ -0,0 +1,67 @@
+---
+title: New Release! Grain v0.4.0 - Cereal
+subtitle: Announcing the release of Grain v0.4.0, Cereal, with NEAR support.
+date: 2021-09-07 12:00:00
+author: Blaine Bublitz
+cover: /blog/photos/cover/marie-bellando-mitjans-cBChXxQqEKM-unsplash.jpg
+coverAttribution: Marie Bellando-Mitjans / Unsplash
+tags: Release, Changelog
+---
+
+After nearly 5 months of hard work, we are excited to finally share Grain v0.4.0 with the world! This release marks the culmination of a bunch of parallel features being built by the core team and a ton of amazing contributions from our community. 🎉
+
+As is typical with these major announcements, I will share some of the highlights we are excited about, but you should check out all of the awesome in the changelog attached to our [tagged GitHub release](https://github.com/grain-lang/grain/releases/tag/grain-v0.4.0).
+
+## NEAR Support
+
+A big portion of the work that went into this release was preparing the compiler to output WebAssembly that will run on the NEAR blockchain. NEAR already supports Rust (compiled to wasm) and AssemblyScript, but any language that targets WebAssembly should be able to output smart contracts that will run on their platform. We needed to make some adjustments to the compiler itself to support intricacies with the platform, but the NEAR Foundation graciously provided us with [a $50,000 grant](https://grain-lang.org/blog/2021/08/16/grain-awarded-50000-grant-from-near-foundation/) to make this work possible!
+
+We have preliminary support for NEAR in this release, but things will get **really** interesting once we write the NEAR SDK for Grain. This is some of our most pressing work now that the release is out—check back soon to start writing smart contracts in Grain. 📈
+
+## Tools
+
+Another important goal for this release was to improve the tooling experience for our end users and for ourselves. Towards this goal, we added two new commands to the Grain CLI: `grain format` and `grain doc`.
+
+### Grain format
+
+Grain format provides an opinionated code formatter tool for Grain source code. This tool was an absolutely herculean effort by Marcus, the core team member we introduced in our last release post and maintainer of the Grain Language Server Protocol.
```

Comment:
```suggestion
Grainformat provides an opinionated code formatter tool for Grain source code. This tool was an absolutely herculean effort by [Marcus](https://github.com/marcusroberts), the core team member we introduced in our last release post and maintainer of the Grain Language Server Protocol.
```
🤷 on the GH profile link; might be nice for us to do for shout-outs

Reply:
I trend towards twitter
```suggestion
Grain format provides an opinionated code formatter tool for Grain source code. This tool was an absolutely herculean effort by [Marcus](https://twitter.com/marcusr), the core team member we introduced in our last release post and maintainer of the Grain Language Server Protocol.
```